Example/AccessibilitySnapshot/SwiftUIViewWithScrollView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright 2020 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A SwiftUI View inside a ScrollView will produce no accessibility elements
+/// for iOS 14.0 and 14.1, for more information see the issue:
+/// AccessibilitySnapshot/issues/33. This seems to be a SwiftUI bug which was
+/// resolved in iOS 14.2.
```

Comment:
nit: Let's format as `cashapp/AccessibilitySnapshot#33` since that's what GitHub will automatically link if you copy it. Also, I think the comment can be above the `@available` to keep the code together.

```suggestion
/// A SwiftUI View inside a ScrollView will produce no accessibility elements for iOS 14.0 and 14.1, for more
/// information see cashapp/AccessibilitySnapshot#33. This seems to be a SwiftUI bug which was resolved in iOS 14.2.
@available(iOS 13.0, *)
```

Reply:
good to know 👌